src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -280,6 +285,10 @@ bool CountryInstance::add_reform(Reform const& new_reform) {
 	}
 }
 
+void CountryInstance::set_strata_tax_rate(Strata const& strata, const StandardSliderValue::int_type new_value) {
```

Comment:
`new_value` is very vague.
`tax_percentage` would be better.

Reply:
I think it's clear enough, after all it's an integer type so it definitely can't be a decimal proportion